### ISSUE TITLE
Atelier Ayesha: The Alchemist of Dusk DX (1152300)

### ISSUE THREAD

**Issue by CartoonFan** [ValveSoftware/Proton#5134]
# Compatibility Report
- Name of the game with compatibility issues: Atelier Ayesha: The Alchemist of Dusk DX
- Steam AppID of the game: 1152300

## System Information
- GPU: NVIDIA Corporation NVIDIA GeForce GTX 960/PCIe/SSE2
- Driver/LLVM version: 4.6.0 NVIDIA 470.56.05
- Kernel version: 5.13.12-arch1-1-bcachefs-git-354570-gbf6289ccf9e9
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/CartoonFan/52c06bf6c536066e06d1450447306cdb
- Proton version:  6.3
## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1152300.log](https://github.com/ValveSoftware/Proton/files/7098870/steam-1152300.log)

## Symptoms 
Game crashes at the start menu with Proton 6.3. The menu's reachable with Proton 5.13, but I haven't done any more testing than that.

## Reproduction

Start the game and wait :)

**Comment by BillFleming** [comment]
FYI the game was fully playable with older 5.X proton such as 5.0-10. (beat story boss at least)
Game also seems to work fine on Proton-5.21-GE-1.
However it crashes on Proton-6.21-GE-2 as well.
This is most likely video/shared resources related regression.
Also crashes on Proton-7.0rc3-GE-1 :-(

**Comment by CartoonFan** [comment]
Yeah, I had the same thing. Similarly to Proton-GE, my build of proton-tkg plays the first few notes of the menu theme, then crashes. Curiously, it doesn't crash when winegstreamer is disabled, so you might very well be on to something.
